Ember-FrontEnd/app/templates/index.hbs

```diff
@@ -1,6 +1,6 @@
 
 <div class="page-title">
-  <h3>Book Recommender</h3>
+  <img src = "/assets/images/logo.png" style = "width:125px;height:125px;">
```

Comment:
The logo looks great! I suggest moving the styling to the app.css file. Something like this...

```
// app.css

...
#logo {
    width: 125px;
    height: 125px;
}
```

```suggestion
  <img src = "/assets/images/logo.png" id= "logo">
```

Reply:
The changes have been made.